buildconfig/version.py.in

```diff
@@ -27,3 +27,46 @@ The python version information should always compare greater than any previous
 releases. (hmm, until we get to versions > 10)
 """
 
+class VersionInfo(object):
```

Comment:
You could simplify this class by using a `namedtuple`. It handles all the accesses and comparisons but it requires an import.

```python
from collections import namedtuple

class VersionInfo(namedtuple('VersionInfo', 'major minor patch')):
    __slots__ = ()

    # Override __repr__ here if you want something other than the default. 
    # Default __repr__ output: "VersionInfo(major=2, minor=0, patch=0)"

    def __str__(self):
        return '{}.{}.{}'.format(*self)
```

Reply:
Ah, I forgot about named tuples. Thanks